### PR TITLE
feat: make layout responsive to improve UX on small screens

### PR DESCRIPTION
**Pull request by marcosvega91**
**Before**
Before
**Now**
Now

### CODE REVIEW

**Comment by smeijer** [comment]
Thanks for this. On mobile, it looks great. Although, this is not quite what I meant with the sketch in #16. This layout will make it unworkable on large screens.

We need a responsive breakpoint, that will apply the layout you now got here, on small screens. And the layout currently in production, on larger screens.

See: https://tailwindcss.com/docs/responsive-design/#targeting-mobile-screens

This is how the deploy preview looks at my screen: https://deploy-preview-16--testing-playground-com.netlify.app/

image

Can you fix it like that?

**Comment by marcosvega91** [comment]
It's a strange behavior because I inspect with chrome dev tools the preview and the styles from TailWinds are not applied. 

**Local**
image
**Deployed**
image

**Comment by smeijer** [comment]
I see. I was having the same issues with the `hover:xxx` class names. Somehow the modifiers with a `:` don't make it in the production bundle.

I see two solutions. 

- find and fix the root cause of that issue, why don't those classes work on production mode? 
- declare some styles in `app.pcss`

**Comment by smeijer** [comment]
Incorrectly auto closed by #19

File: src/components/App.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ function App() {
       </div>
 
       <div className="space-y-8 px-8 pb-8">
-        <div className="editor">
+        <div className="editor lg:grid-cols-2 sm:grid-cols-1">
```
Comment (smeijer) [inline src/components/App.js:43]:
```suggestion
        <div className="editor grid-cols-1 md:grid-cols-2">
```

The `sm:` prefix isn't needed. That should be the default. And I believe `md` is large enough for 2 columns. What do you think?

Comment (marcosvega91) [reply src/components/App.js:44]: md screen I think is too small for 2 columns. Look at the image below


image

Comment (smeijer) [reply src/components/App.js:44]: I like to believe that most developers have larger monitors than the average consumer:

For me having 1 column on large, wouldn't be a good thing for user experience:

image

Having 2 columns on large, and 1 on medium or smaller, would be a good thing though:

image

image

We can make `grid-cols-1 md:grid-cols-2` the default, and add a toggle later that allows users to switch between these two layouts manually?

Comment (marcosvega91) [reply src/components/App.js:44]: Yes could be great

**Comment by smeijer** [comment]
Thanks :tada:

**Pull request merged**